### PR TITLE
[Naitei] ft:View_Details_Bill_Admin

### PR DESCRIPTION
**Pull request by ThaiKhacDuoc**
## Related Tickets
- ticket redmine
https://edu-redmine.sun-asterisk.vn/issues/78426?issue_count=58&issue_position=34&next_issue_id=78425&prev_issue_id=78427
## WHAT (optional)
- Change number items `completed/total` in admin page.

## HOW
- I edit js file, inject not_vary_normal items in calculate function.

## WHY (optional)
- Because in previous version - number just depends on `normal` items. But in new version, we have `state` and `confirm_state` depends on both `normal` + `not_normal` items.

## Evidence (Screenshot or Video)
image

### CODE REVIEW

**Comment by ThaiKhacDuoc** [comment]
ready

File: app/controllers/admin/bills_controller.rb [hunk 1]
```diff
@@ -3,4 +3,8 @@ def index
     @pagy, @bills = pagy(Bill.order(created_at: :desc),
                          items: Settings.page_size)
   end
+
+  def show
+    @bill = Bill.find(params[:id])
```
Comment (ghost) [inline app/controllers/admin/bills_controller.rb]: xử lý trường hợp không tìm thấy

**Review by ghost — COMMENTED** [review]
cmt

**Comment by ThaiKhacDuoc** [comment]
ready

**Comment by ThaiKhacDuoc** [comment]
ready

File: app/controllers/admin/bills_controller.rb [hunk 2]
```diff
@@ -3,4 +3,11 @@ def index
     @pagy, @bills = pagy(Bill.order(created_at: :desc),
                          items: Settings.page_size)
   end
+
+  def show
+    @bill = Bill.find_by(id: params[:id])
+
+    flash.now[:danger] = t("bills.not_found")
+    redirect_to admin_bills_path unless @bill
```
Comment (ghost) [inline app/controllers/admin/bills_controller.rb]: Viết như này thì tìm thấy vẫn hiện flash à

**Review by ghost — COMMENTED** [review]
cmt

**Comment by ThaiKhacDuoc** [comment]
ready

**Pull request merged**